integration-tests/src/test_loop/tests/resharding_v3.rs

```diff
@@ -777,16 +941,12 @@ fn test_resharding_v3_base(params: TestReshardingParameters) {
         // Give enough time to produce ~7 epochs.
         Duration::seconds((7 * params.epoch_length) as i64),
     );
+    let client = &test_loop.data.get(&client_handles[0]).client;
+    trie_sanity_check.check_epochs(client);
```

Comment:
Would it be possible to call `check_epochs` for all clients, at least when single shard tracking is enabled?

Reply:
Oh we actually kind of do check the results for all clients, because we have the details stored for each account ID in `trie_sanity_check`, but we just pass in a `client` here to get the info on what epoch IDs there were